src/Chapter1.hs

```diff
@@ -620,7 +626,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 n =
+    let lastNumber = rem n 10
+        numberLeft = div n 10
```

Comment:
`rem` usually goes in pair with `quot` (instead of `div`), and `div` with `mod` (instead of `rem`). 

One hint to make your idea a bit shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(d, n) = divMod x 10
```

You can see how we could pattern match on the pair 🙂 

Also, note, that it won't work on negative input.